charts/drax/values.yaml

```
@@ -49,6 +61,23 @@ global:
   #
   accelleranProduct: "P5G"
 
+  # Alerting Configuration
+  #   Description: This section allows you to configure the alerting settings for the Accelleran dRAX.
+  #
+  #   Parameters:
+  #     - `enabled`:      A boolean flag to enable or disable alerting. When set to `true`, alerting is activated;
+  #                       when set to `false`, alerts are not provisioned.
+  #     - `slackToken`:   Required: The token used to configure the Slack contact point for alert notifications.
+  #                       This token allows alerts to be sent to the specified Slack channel.
```

The slack channel is not specified here.

```suggestion
  #                       This token allows alerts to be sent via Slack.
```